hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/MetricsWAL.java

```diff
@@ -91,4 +92,12 @@ public void logRollRequested(WALActionsListener.RollRequestReason reason) {
         break;
     }
   }
+
+  public void postLogRoll(Path oldPath, Path newPath) throws IOException {
```

Comment:
nit: remove `throws IOException`?

Reply:
Thank you for the review. Fixed in latest commit.